wip/services/exec/store.go

```diff
@@ -80,8 +91,20 @@ func (p postgresStore) Get(id string) (types.Exec, error) {
 		}
 		return types.Exec{}, err
 	}
+	keyRefs, err := db.Q.ExecSSHKeysGetByExecID(ctx, id)
+	if err != nil {
+		return types.Exec{}, err
+	}
+	keyIDs := db.MapStrings(keyRefs, func(key db.UnweaveExecSshKey) string {
```

Comment:
It's not clear what this function does by it's name `MapStrings`

Reply:
Map is a specialist term in all of functional programming to refer to this kind of operation. It simply refers to a transformation you do across a list that outputs another. If you’re not familiar with FP I can understand why this would be confusing so let’s avoid this naming in our codebase

Reply:
I understand the map operation but seeing it it the wild called just `MapStrings` doesn't make it clear what's happening. I renamed it to `MapToStrings` and added a doc string.

Reply:
That's fair!